### ISSUE TITLE
Generator overlapping cards

### ISSUE THREAD

**Issue by TheValko** [mephitrpg/rpg-cards#100]
When i generate a list of cards the generator starts to overlap cards and display them incorrectly. It displays the same way in the print screen as well

**Comment by crobi** [comment]
This happens when you try to fit too many cards on one page. What are your settings (card size, page size, number of cards per page)?

**Comment by TheValko** [comment]
I am using the standard default setting of A4 pages, 2.5 x 3.5 cards and 3x3 per page. about half of my list shows up perfectly then it messes up.

**Comment by ja-liviero** [comment]
I'm seeing the same issue.
* US Letter
* 2.5 x 3.5 card size
* 3x3 per page
* 27 cards total (so exactly 3 full pages)

**Comment by ja-liviero** [comment]
Managed to fix it for myself.  I found an un-closed italics tag (`<i><i>` instead of `<i></i>`).

A bit out of scope here, but what about adding `_words_` and `**words**` as formatting keys for italics and bold respectively, to avoid this hiccup?  That method is already commonly used in many places, including this here.

**Comment by TheValko** [comment]
Yep, that's what was wrong with mine as well. Nice catch, thanks a lot!

**Comment by crobi** [comment]
Good to hear you found the issue. If you're up to it, feel free to propose a new feature (either validating html tags or implementing `_word_`, `**word**`, `*word*`, and `~word~` formatting) in a new issue (or even better, implement it and post a PR).

**Comment by ja-liviero** [comment]
I'd be happy to submit a PR, but it might be a little while before I get to it.  Thanks!
